vulnerability-disclosure-policy.md

```diff
@@ -67,3 +67,11 @@ Reports should include:
 Please keep your vulnerability reports current by sending us any new information as it becomes available.
 
 We may share your vulnerability reports with [US-CERT](https://www.us-cert.gov/ais), as well as any affected vendors or open source projects.
+
+### Coordinated Disclosure
+
+TTS is committed to patching vulnerabilities within 90 days or less, and disclosing the details of those vulnerabilites once patches are available. We believe that public disclosure of vulnerabilities is an essential part of the vulnerability disclosure process, and that one of the best ways to make software better is to enable everyone to learn from each other's mistakes.
+
+At the same time, we believe that disclosure in absence of a readily available patch tends to increase risk rather than reduce it, and so we ask that you refrain from sharing your report with others while we work on our patch.  If you believe there are others that should be informed of your report before the patch is available, please let us know so we can make arrangements.
+
+We may want to coordinate an advisory with you to be published simultaneously with the patch, but you are also welcome to self-disclose if you prefer.  By default, we prefer to disclose everything, but we will never publish information about you or our communications with you without your permission.  In some cases, we may also have some sensitive information that should be redacted, and so please check with us before self-disclosing.
```

Comment:
I think the "please check with us before self-disclosing" could be read the wrong way, that we're asking to approve self-disclosure. We can't exactly do that (well, we can _ask_, but people may not listen). Instead I think we should make clear what sort of information we'll redact (PII, and information about other unfixed issues, I'm guessing) and suggest that self-disclosers do the same.

Reply:
@jacobian Suggested phrasing to minimize risk of being mis-read?  I doubt we can know in advance all of the that information will be sensitive.

Reply:
How about just: "We'll redact sensitive information such as a PII and information about other vulnerabilities and request that you do as well."